app/Http/Requests/VisitTransfer/ApplicationCompleteRequest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Http\Requests\VisitTransfer;
+
+use App\Models\VisitTransfer\Application;
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Support\Facades\Gate;
+
+class ApplicationCompleteRequest extends FormRequest
+{
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'complete_staff_note' => 'nullable|string',
```

Comment:
Do we want/need a length limit here? Because any single letter on its own would suffice using this validation. e.g `min:25`